rules-tests/DeadCode/Rector/ClassMethod/RemoveUselessReturnTagRector/Fixture/remove_return_construct.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\ClassMethod\RemoveUselessReturnTagRector\Fixture;
```

Comment:
move the fixture under new namespace directory `RemoveUselessVoidReturnFromDocblockVoidMagicMethodsRector`